code/modules/mod/mod_theme.dm

```diff
@@ -598,6 +598,7 @@
 	slowdown_inactive = 1
 	slowdown_active = 0.5
 	allowed_suit_storage = list(
+		/obj/item/gun, // SKYRAT EDIT - GUN IN SUIT SLOT
```

Comment:
Try getting the gun thing added upstream first.

Reply:
Discussed with Zonespace in DMs. I'm waiting on their final thoughts for now.

Reply:
Overall opinion on this is that I'd rather not allow security to carry two long-arms on them, one of which may as well be invisible @KathyRyals